sbpy/photometry/core.py

```diff
@@ -1046,7 +1191,56 @@ def fit_deriv(ph, h, g):
         else:
             p1 = 0.9529
             p2 = -0.6125
-        ddg = 1.085736205*((phi3-phi1)*p1+(phi3-phi1)*p2)/dom
+        ddg = 1.085736205*((phi3-phi1)*p1+(phi3-phi2)*p2)/dom
+        return [ddh, ddg]
+
+    def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
+        return OrderedDict([('H', outputs_unit['y']),
+                            ('G12', u.dimensionless_unscaled)])
+
+
+class HG12_Pen16(HG12):
```

Comment:
is this the latest-greatest version of G12? If so, should we make it the default?

Reply:
I don't know.  I think the difference is that this new version doesn't have discontinuity in the `G12` parameter and its 1st derivative.  But I have no idea which one is "more official" or "better".  Let me ask Antti.